__tests__/app.test.js

```diff
@@ -426,3 +426,151 @@ describe("DELETE /api/comments/:comment_id", () => {
       });
   });
 })
+describe("/api", () => {
+  test("GET 200: gets endpoints json file", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({body}) => { 
+        const output = {
+          "GET /api": {
+            "description": "serves up a json representation of all the available endpoints of the api"
+          },
+          "GET /api/categories": {
+            "description": "serves an array of all categories",
+            "queries": [],
+            "exampleResponse": {
+              "categories": [
+                {
+                  "description": "Players attempt to uncover each other's hidden role",
+                  "slug": "Social deduction"
+                }
```

Comment:
Rather that having to write this all out in the test file (which makes for quite a lengthy test) I think a good alternative would be to simply `require` in the `endpoints.json` file that you use for this endpoint! That way if you ever change the file, the test won't need to change in order to update it.